lib/verbalize/action.rb

```diff
@@ -17,9 +17,20 @@ def self.included(target)
 
     module ClassMethods
       def input(*required_keywords, optional: [])
+        @inputs = required_keywords
+        @optional_inputs = optional
+
         class_eval Build.call(required_keywords, Array(optional))
       end
 
+      def inputs
```

Comment:
Also @taylorzr I don't agree here with separating `input` and `inputs` at least for the purpose of accessing the full parameter list.

There is no contextual indicator of the difference between these two methods.  Conceptually, the original implementation made more sense, if we ignore that it returned less than the full set of inputs when used as a getter.

While it is generally good to have methods with individual responsibilities since it simplifies the logic of the accessor method itself, it is a common pattern to use the same method as a setter when params are given and getter when there are none.  It eliminates the ambiguity that currently exists between `input` and `inputs` which do not conform to either the setter-as-accessor pattern nor the typical `getter` / `setter=` accessor pattern we are used to.

TL;DR - if we are going to have a getter method for accessing the full input list directly, the original implementation here seems best (modified to return required+optional parameters).